akka-sample-camel-java/README.md

```diff
@@ -1,12 +1,12 @@
-This tutorial contains 3 samples of [Akka Camel](http://doc.akka.io/docs/akka/2.5.0/java/camel.html).
+This tutorial contains 3 samples of [Akka Camel](http://doc.akka.io/docs/akka/current/java/camel.html).
```

Comment:
I think I'd like to promote using 'current' - when we release 2.6 we should also update the samples accordingly anyway :).